construction_impl.go

```diff
@@ -371,6 +371,118 @@ func (r RosettaConstructionFilecoin) ConstructSwapAuthorizedParty(request *SwapA
 	return string(tx), nil
 }
 
+func (r RosettaConstructionFilecoin) ConstructRemoveAuthorizedParty(request *RemoveAuthorizedPartyRequest, destinationActorId cid.Cid) (string, error) {
+	to, err := filAddr.NewFromString(request.Multisig)
+	if err != nil {
+		return "", err
+	}
+
+	from, err := filAddr.NewFromString(request.From)
+	if err != nil {
+		return "", err
+	}
+
+	gasfeecap, err := types.BigFromString(request.Metadata.GasFeeCap)
+	if err != nil {
+		return "", err
+	}
+
+	gaspremium, err := types.BigFromString(request.Metadata.GasPremium)
+	if err != nil {
+		return "", err
+	}
+
+	gaslimit := request.Metadata.GasLimit
+
+	decreaseParams := request.Params.Decrease
+
+	toRemoveParams, err := filAddr.NewFromString(request.Params.ToRemove)
+	if err != nil {
+		return "", err
+	}
+
+	var methodNum abi.MethodNum
+	var serializedParams []byte
+
+	switch destinationActorId {
+	case builtinV1.MultisigActorCodeID:
+		methodNum = builtinV1.MethodsMultisig.Propose
+
+		removeSignerParams := &multisigV1.RemoveSignerParams{
+			Signer:   toRemoveParams,
+			Decrease: decreaseParams,
+		}
+		bufRemoveSigner := new(bytes.Buffer)
+		err = removeSignerParams.MarshalCBOR(bufRemoveSigner)
+		if err != nil {
+			return "", err
+		}
+
+		params := &multisigV1.ProposeParams{
+			To:     to,
+			Value:  types.NewInt(0),
+			Method: builtinV1.MethodsMultisig.SwapSigner,
```

Comment:
Should be calling `RemoveSigner`

Reply:
good 👁️ s. @raynaudoe can we fix it?

Reply:
oopsss  :+1: 
we definitely need a bit more coverage here